src/components/Accordion.tsx

```diff
@@ -0,0 +1,44 @@
+import { Css, IconButton, useTestIds } from "src/index";
+import { ReactNode, useState } from "react";
+
+export interface AccordionProps {
+  title: string;
+  children: ReactNode;
+  disabled?: boolean;
+  defaultExpanded?: boolean;
+  bottomSection?: boolean;
```

Comment:
**thought:** This `bottomSection` property makes me think we can maybe do this smarter. I don't like the idea of a user having to define that this Accordion is the last Accordion in my list. Maybe instead what we can do is create another component in this PR, for instance `<AccordionList accordions=[[] as AccordionProps] />`. Then this component can do the proper styling for a list of Accordions by defining the `bottomSection`

Reply:
Another cool thing with an AccordionList is that we could provide a prop that can allow the list to only allow one Accordion open at a time vs multiple. Could be neat.

Reply:
This also makes me wonder, if you have only one Accordion, then how should it look? Should a single accordion include the bottom border? I'll ping the Figma again.